js/dropdown.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+    $('[data-toggle="dropdown-with-sub-menu"]').click(function(event) {
+        if ($(window).width() > 1024) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/eyinwag/calico/pullRequest?prid=8501747)
```suggestion
        if ($(window).width() > 1024) {return;}
```